.travis.yml

```diff
@@ -24,18 +24,17 @@ before_install:
   - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 762E3157
   # Loop until update succeeds (timeouts can occur)
   - travis_retry $(! sudo apt-get -qq update 2>&1 | grep Failed)
-  - sudo apt-get install -y gnupg2 git
-  - sudo apt-get install -y golang-1.14
-  - export PATH="/usr/lib/go-1.14/bin:$PATH"
-  - export GOROOT=/usr/lib/go-1.14
-  - go version
+  - sudo apt-get install -y gnupg2 git curl
+  # Install Helm Binary
+  - curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
```

Comment:
should we be pinning the version of helm3 that gets downloaded? I believe if I'm reading the `get_helm.sh` script correctly, we could technically set a `DESIRED_VERSION` to control the helm3 version being downloaded instead of defaulting to latest.